src/snackbar.jsx

```diff
@@ -227,6 +224,8 @@ const Snackbar = React.createClass({
   timerOneAtTheTimeId: undefined,
 
   componentClickAway() {
+    if (!this.timerTransitionId) return;
```

Comment:
Don't execute clickAway callback if the timerTransitionId is truthy, such as not 0 (or in other words when animation is still running).